tests/integration/test_wazuh_db/data/agent_messages.yaml

```diff
@@ -0,0 +1,69 @@
+---
+-
+  name: "Agents' CVEs table: vuln_cves "
+  description: "Checks the commands insert and clear"
+  test_case:
+  -
+    input: 'agent 000 vuln_cve insert {"name":"test_package","version":"1.0","architecture":"x86","cve":"1001"}'
```

Comment:
Although the database accepts any string as a CVE, I would use a string with the format of a CVE. Something like `CVE-XXXX-YYYY` where `XXXX` is a year, and `YYYY` just a number.

Reply:
Done!